src/extension.ts

```diff
@@ -51,27 +65,48 @@ export function activate(context: vscode.ExtensionContext) {
     l2Version === undefined ||
     semver.lt(l2Version, MIN_VERSION_TO_CHECK)
   ) {
-    // The L2 version, if less than v1.5.0, does not support the `l2 -e <filepath>` feature.
-    // Therefore, we use the extension to fetch variables from the l2.env file for suggestions.
+    /**
+     * The L2 version, if less than v1.5.0, does not support the `l2 -e <filepath>` feature.
+     * Therefore, we use the extension to fetch variables from the l2.env file for suggestions.
+     */
     getDotENVS();
     suggestEnvVariables = suggestENVSFromDotEnv();
   } else {
-    // L2 versions greater than or equal to v1.5.0 support the `l2 -e <filepath>` feature.
-    // Therefore, we fetch variables from both the l2.env and l2config.env files for suggestions using this command.
-    suggestEnvVariables = lama2RegisterCompletionItemProvider();
+    /**
+     * L2 versions greater than or equal to v1.5.0 support the `l2 -e <filepath>` feature.
```

Comment:
No need to retain this as well. Only LSP going forward.

Reply:
Cool